### PR TITLE
[IMP] website: introduce `s_key_images` snippet

### PR DESCRIPTION
**Pull request by Brieuc-brd**
This commit adds the new `s_key_images` snippet.

task-4105059
Part of task-4077427

Requires:
- https://github.com/odoo/design-themes/pull/878

Capture d’écran 2024-08-22 à 10 19 22

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Review by chgo-odoo — APPROVED** [review]
LGTM, thanks ! 👍

**Comment by Brieuc-brd** [comment]
- Added custom texts for themes
- Reviewed images of Loftspace and Threehouse
- Rebased

**Review by bvr-odoo — CHANGES_REQUESTED** [review]
@Brieuc-brd Here is my review, thanks for the work

File: addons/website/views/snippets/s_key_images.xml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_key_images" name="Key Images">
+    <section class="s_key_images pt72 pb72">
+        <div class="container">
+            <div class="row s_nb_column_fixed">
+                <div class="col-lg-12 pb32">
+                    <h2 class="display-5">What we propose to our customers</h2>
```
[the remark this answers is not in the document]
Comment (Brieuc-brd) [reply addons/website/views/snippets/s_key_images.xml]: Indeed 🙁 Thanks for creating a PR for that 👍 
I've also started checking all my Open PRs to solve this issue asap :)
Here it's now fixed ✅

File: addons/website/views/snippets/s_key_images.xml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+                    <p class="lead">Dive deeper into our company’s abilities.</p>
+                </div>
+                <div class="col-6 col-lg-3">
+                    <p class="h1-fs">01</p>
+                    <img src="/web/image/website.s_key_images_default_image_1" class="img img-fluid rounded" alt="" style="width: 100% !important;"/>
```
[the remark this answers is not in the document]
Comment (Brieuc-brd) [reply addons/website/views/snippets/s_key_images.xml]: Good catch, thanks 👍 
Fixed, I also adapted the themes as well ✅

**Comment by Brieuc-brd** [comment]
Hey @bvr-odoo 👋 
Thanks for your review, I've updated the branches and left some comments.
Here's the diff : 
- COM: https://github.com/odoo/odoo/compare/5a480662fffca18f8fe7b524353a3b06fc62cee3..17dd2cf22bb078f497fb6a1090f3f629eb084c63
- DT: https://github.com/odoo/design-themes/compare/77284af81391469daeedeb3bbdf9c1b5eb710fa5..bd6b9149327b7a7a11744b38c07d8c4e63b23411

I'll rebase to solve conflicts once these PRs will be ready to be r+ 🙂

**Comment by bvr-odoo** [comment]
Hi @Brieuc-brd , LGTM :+1:  could you rebase to have both runbots green and without conflicts. Thanks :pray:

**Comment by Brieuc-brd** [comment]
Hey @bvr-odoo 👋, done ✅

**Comment by bso-odoo** [comment]
@robodoo delegate=bvr-odoo

**Review by bvr-odoo — APPROVED** [review]
LGTM :+1:

**Comment by bvr-odoo** [comment]
@Brieuc-brd runbot is red :/

**Comment by Brieuc-brd** [comment]
@bvr-odoo It's now green 🟢

**Comment by bvr-odoo** [comment]
@robodoo r+

**Pull request closed without merging**